website/moneybirdsynchronization/services.py

```diff
@@ -18,18 +35,10 @@ def create_or_update_contact(member):
     moneybird = get_moneybird_api_service()
 
     if moneybird_contact.moneybird_id is None:
-        # Check if the contact already exists in Moneybird somehow.
-        response = moneybird.get_contact_by_customer_id(member)
-        if response is None:
-            # Push the contact to Moneybird.
-            response = moneybird.create_contact(moneybird_contact.to_moneybird())
-            moneybird_contact.moneybird_id = response["id"]
-        else:
-            # Link the existing moneybird contact. Then update the data.
-            moneybird_contact.moneybird_id = response["id"]
-            response = moneybird.update_contact(
-                moneybird_contact.moneybird_id, moneybird_contact.to_moneybird()
-            )
+        # Push the contact to Moneybird.
+        response = moneybird.create_contact(moneybird_contact.to_moneybird())
+        moneybird_contact.moneybird_id = response["id"]
+        moneybird_contact.moneybird_sepa_mandate_id = response["sepa_mandate_id"]
```

Comment:
If there is no SEPA mandate, the response is empty and Django processes this as an empty string. However there is are null=true and unique=true constraints on the model field.
This means that Django checks if "" == "" which is true, thus the constraint fails ([source](https://stackoverflow.com/questions/17257031/django-unique-null-and-blank-charfield-giving-already-exists-error-on-admin-p)) This can be solved by doing your own uniqueness checks.

Reply:
I made it cast to None